### PR TITLE
0.7.0 release

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

**Comment by adam-fowler** [comment]
@fwcd , @smumriak Do you want your correct email addresses in the `CONTRIBUTORS.txt` file

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
...
+### Added
+
+- Support multiple workspace feature of SourceKit-LSP that comes with Swift 5.7.
+- Tasks for SwiftPM plugins.
+
+### Changed
+
+- When running with Swift 5.7 or later, migrate to using official implementation of Inlay Hints.
+- Cleanup Swift Task implementation, remove `command` property as it is always `swift`.
+- Show no tests messages instead of an error message in Test Explorer when a package has no tests.
```
Comment (0xTim) [inline CHANGELOG.md:18]:
```suggestion
- Show "no tests" message instead of an error message in Test Explorer when a package has no tests.
```

**Review by 0xTim — APPROVED** [review]
One nit but LGTM

**Comment by fwcd** [comment]
Looks good to me as-is, I prefer the GitHub address, but if this is important, you can also use my other committer email.

**Comment by smumriak** [comment]
Hey @adam-fowler, you can use either github email or sergiy.mumryak@gmail.com for me. Your choice

**Comment by adam-fowler** [comment]
> Looks good to me as-is, I prefer the GitHub address, but if this is important, you can also use my other committer email.

No requirement on including your email. I was just giving you the option if you desired it. Don't be offended if I ask you this again in a couple of months. It just means I forgot you'd rather keep the GH address.

**Pull request merged**